bump.py

```diff
@@ -33,22 +30,41 @@ def run(args):
         bug += 1
 
     version = f"{major}.{minor}.{bug}{version_post}"
-    with open(args.version_file, "w") as f:
+    with open("VERSION", "w") as f:
         f.write(version)
 
-    if args.version_file == "VERSION":
-        # replace node binding toml version as well
-        with open("nucliadb_node_binding/Cargo.toml", "r") as f:
-            cargo = f.read()
+    # replace node binding toml version as well
+    with open("nucliadb_node_binding/Cargo.toml", "r") as f:
+        cargo = f.read()
 
-        new_cargo = []
-        for line in cargo.splitlines():
-            if line.startswith("version ="):
-                line = f'version = "{version}"'
-            new_cargo.append(line)
+    new_cargo = []
+    for line in cargo.splitlines():
+        if line.startswith("version ="):
+            line = f'version = "{version}"'
+        new_cargo.append(line)
 
-        with open("nucliadb_node_binding/Cargo.toml", "w") as f:
-            f.write("\n".join(new_cargo))
+    with open("nucliadb_node_binding/Cargo.toml", "w") as f:
+        f.write("\n".join(new_cargo))
+
+    # go through each requirements.txt and update the version to the new bump
+    for req_filepath in (
+        "nucliadb/requirements.txt",
+        "nucliadb_utils/requirements.txt",
+        "nucliadb_sdk/requirements.txt",
```

Comment:
is nucliadb_telemetry missing?

Reply:
It doesn't depend on anything else so we don't need to update any of the dependencies!